### ISSUE TITLE
Dashboard - Unwanted movement of widgets and new functions on mobile

### ISSUE THREAD

**Issue by aleadco** [nextcloud/server#24404]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
The movement of the widgets of the dashboard application generates a bad user experience on a touch device because when you try to slide the screen, you tend to select and move the widgets.

**Describe the solution you'd like**
I think that a function should be incorporated that avoids this edition unless the user presses the personalization button. Also incorporate in the global administration options that make it possible to manage which application is enabled by default for all users, what order the widgets should have and another option that allows administrators to decide whether users can make modifications.

**Describe alternatives you've considered**

**Additional context**
![Screenshot_20201126-214711_Chrome](https://user-images.githubusercontent.com/35962168/100405410-87836c80-3031-11eb-94d1-ec29f94ffc09.jpg)

**Comment by szaimen** [comment]
I can reproduce this on NC21.0.2

**Comment by szaimen** [comment]
cc @nextcloud/designers

**Comment by raimund-schluessler** [comment]
I think it would be enough to just delay the drag and drop event. This way one can still scroll, even when touching the widget, but also still drag it just by pressing the widget for longer. Works very well for the Tasks app for example. And since Dashboard and Tasks uses the same library, the fix should be very easy and straightforward. Just add `swapThreshold: 0.30, delay: 500, delayOnTouchOnly: true, touchStartThreshold: 3` (see https://github.com/nextcloud/tasks/blob/master/src/components/TaskDragContainer.vue#L26) to the draggable configuration of the widget here: https://github.com/nextcloud/server/blob/master/apps/dashboard/src/App.vue#L12

**Comment by szaimen** [comment]
@raimund-schluessler do you mind creating a PR with that fix? Would be great! Thank you! :)
